cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -5,15 +5,22 @@
 @rfm.required_version('>=2.14')
 @rfm.simple_test
 class StreamTest(rfm.RegressionTest):
+    """This test checks the stream test:
+       Function    Best Rate MB/s  Avg time     Min time     Max time
+       Triad:          13991.7     0.017174     0.017153     0.017192
+    """
     def __init__(self):
         super().__init__()
         self.descr = 'STREAM Benchmark'
         self.exclusive_access = True
-        # All available systems are supported
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
                               'kesch:cn', 'kesch:pn', 'leone:normal']
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
-                                    'PrgEnv-intel', 'PrgEnv-pgi']
+        if self.current_system.name in {'dom', 'daint', 'kesch', 'leone'}:
+            self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-gnu',
+                                        'PrgEnv-intel', 'PrgEnv-pgi']
+        else:
+            self.valid_prog_environs = ['*']
```

Comment:
As discussed with @rsarm this is not needed. When running on an unknown system when running as follows should be enough:

```
reframe --system=mysys -p myenv --skip-system-check --skip-prgenv-check ...
```